testsuite/features/secondary/min_migrate_to_trad.feature

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# This test case does not test a valid scenario. In SUMA you normally do not
+# delete a traditional client and then register it again as Salt minion. You
+# would always just bootstrap the traditional client as a Salt minion again
+# and Salt would recognize that it is a traditional client and will take the
+# the necessary steps to migrate it
+
+Feature: Migrate a Salt minion into a traditional client
+  As an authorized user
+  I want to migrate these Salt minions to traditional clients and have everything as before
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Migrate a SLES client into a Salt minion
+    When I follow the left menu "Systems > Bootstrapping"
+    And I enter the hostname of "sle_client" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-SUSE-KEY-x86_64" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+
+  Scenario: Wait until the Salt minion appears
```

Comment:
Let's combine these two?

Reply:
Fixed.

Reply:
I still see both scenarios separated, but it's fine, we can keep like that.